src/sentry/api/endpoints/organization_dashboards.py

```diff
@@ -203,17 +217,14 @@ def handle_results(results):
             serialized.extend(serialize(dashboards, request.user, serializer=list_serializer))
             return serialized
 
+        render_pre_built_dashboard = True
+        if features.has("organizations:dashboards-favourite", organization, actor=request.user):
+            if filter_by and filter_by == "onlyFavorites" or pin_by and pin_by == "favorites":
+                render_pre_built_dashboard = False
```

Comment:
In the case where a user _does_ have the feature flag and we rollout the favourite UI, then this is always going to get tripped to `False`, right? I wonder if we should put prebuilt dashboards at the beginning of the "other" dashboards if it exists and make it unselectable

The reason I say this is because if a user doesn't have any dashboards yet, I think this means we won't show the prebuilt dashboard so their first experience is no dashboard at all. Let's think about it but this change is good right now